connect/src/spirc.rs

```diff
@@ -137,6 +138,7 @@ pub struct SpircLoadCommand {
 const CONTEXT_FETCH_THRESHOLD: u32 = 5;
 
 const VOLUME_STEP_SIZE: u16 = 1024; // (u16::MAX + 1) / VOLUME_STEPS
+const VOLUME_UPDATE_DELAY_MS: u64 = 2000;
```

Comment:
absolutly, sometimes forget that some structs can still be `const`, because `Vec` always doesn't work with const or static. Will adjust  it after we decide if we keep the delay or remove it^^